### PR TITLE
Add com.voxdsp.Snowling

### PR DESCRIPTION
**Pull request by BeverlyCode**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

File: com.voxdsp.Snowling.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+command: snowling
+rename-icon: snowling
+rename-desktop-file: snowling.desktop
+rename-appdata-file: snowling.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
```
Comment (bbhtt) [inline com.voxdsp.Snowling.yaml:14]: Why do you need this?

**Comment by hfiguiere** [comment]
bot, build com.voxdsp.snowling

**Comment by hfiguiere** [comment]
bot, build com.voxdsp.Snowling

**Comment by hfiguiere** [comment]
```
]2;flatpak-builder: Installing snowlingRunning: cc main.c -lSDL2 -lGLESv2 -lEGL -Ofast -lm -o snowling
FB: Running 'flatpak build --die-with-parent --env=FLATPAK_BUILDER_BUILDDIR=/run/build/snowling --nofilesystem=host:reset --filesystem=/srv/buildbot/worker/build-x86_64-2/build/.flatpak-builder/build/snowling-1 --bind-mount=/run/build/snowling=/srv/buildbot/worker/build-x86_64-2/build/.flatpak-builder/build/snowling-1 --build-dir=/run/build/snowling --bind-mount=/run/ccache=/srv/buildbot/worker/build-x86_64-2/build/.flatpak-builder/ccache --unshare=network --env=SOURCE_DATE_EPOCH=1700090908 '--env=CFLAGS=-O2 -pipe -g -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer ' '--env=CXXFLAGS=-O2 -pipe -g -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer ' '--env=LDFLAGS=-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed ' --env=CCACHE_DIR=/run/ccache/disabled --env=PATH=/app/bin:/usr/bin --env=LD_LIBRARY_PATH=/app/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig --env=FLATPAK_BUILDER_N_JOBS=64 /srv/buildbot/worker/build-x86_64-2/build/.flatpak-builder/rofiles/rofiles-RzH3e9 /bin/sh -c 'cc main.c -lSDL2 -lGLESv2 -lEGL -Ofast -lm -o snowling'' on host
main.c:15:10: fatal error: esAux4.h: No such file or directory
   15 | #include "esAux4.h"
      |          ^~~~~~~~~~
compilation terminated.
FB: host_command_exited_cb 2624127 256
```

**Comment by bbhtt** [comment]
bot, build com.voxdsp.Snowling

**Comment by bbhtt** [comment]
Seems fine

**Pull request closed without merging**